### ISSUE TITLE
Incompatibility issue with pipette

### ISSUE THREAD

**Issue by badcrave** [tom5454/Toms-Storage#40]
I am using the mod “pipette” which adds three simple pipes to move items. We are also using expanded storage but that doesn’t seem to be the issue. A player tried to connect pipes to an inventory connector, not really knowing what they were doing. The player also doesn’t know how to use pipette and so they were using all normal pipies.

Depending on how the pipes are connected to the inventory connector, if there is no inventories (beside the pipes) touching the inventory connector, and a player tries to put an item into the storage terminal, it will let them. Upon exiting and reentering the storage terminal , the items will no longer be there. The items are not moved into any chest by the pipes and they aren’t found when destroying the pipes. The items are essentially lost. This seems to happen whether the pipes are leading into or out of the connector. The player did not use an extractor pipe when taking items out of the inventory connector, I will test that later.

If there is a chest touching the inventory connector, then placing items in the storage terminal works, and they can still be found in the terminal later, but they aren’t found in the connected chests. Destroying the pipe above the connector, which was pointing into the connector, will actually drop the items. This seems like the intended behavior. We tested this only with the pipes going into the inv connector, I think if we tested the pipies leading out of the connector it would have sent the items down the pipe, into a chest. I haven’t tested. 

Basically, the issue is that the items are getting deleted if no other chests are present next to the inventory connector

**Comment by FIamey** [comment]
Uploading 2021-05-13_14-26-14_Trim.mp4…

E_

**Comment by FIamey** [comment]
https://user-images.githubusercontent.com/56524051/118203944-1d1d8780-b42b-11eb-9920-71e909627999.mp4

**Comment by badcrave** [comment]
Here is my best attempt at summarily demonstrating the bug.

Video = https://www.dropbox.com/s/liysnfjhqb0uzfa/2021-05-13%2023-49-28_Trim%20%282%29.mp4?dl=0
